org.musicbrainz.Picard.appdata.xml

```diff
@@ -0,0 +1,73 @@
+  <name>Picard</name>
+  <summary>MusicBrainz's music tagger</summary>
+
+  <description>
+    <p>
+      Do you need to clean up your music library? Picard is an open-source
+      cross-platform music tagger by MusicBrainz. It has the ability to
+      identify audio files without any metadata.
+    </p>
+    <p>
+      A variety of plugins are available and you can even write your own.
+      Picard supports a wide range of audio formats and can also lookup an
+      entire CD for you.
+    </p>
+    <p>
+      Tagging audio files has never been easier.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">org.musicbrainz.Picard.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">https://www.daitauha.fr/static/screenshots/picard/main.png</screenshot>
+  </screenshots>
+
+  <url type="homepage">https://picard.musicbrainz.org/</url>
+  <url type="bugtracker">https://tickets.metabrainz.org/projects/PICARD/issues/</url>
+  <url type="faq">https://picard.musicbrainz.org/docs/faq/</url>
+  <url type="help">https://community.metabrainz.org/c/picard</help>
+
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="violence-desecration">none</content_attribute>
+    <content_attribute id="violence-slavery">none</content_attribute>
+    <content_attribute id="violence-worship">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="sex-homosexuality">none</content_attribute>
+    <content_attribute id="sex-prostitution">none</content_attribute>
+    <content_attribute id="sex-adultery">none</content_attribute>
+    <content_attribute id="sex-appearance">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
+    <content_attribute id="language-humor">none</content_attribute>
+    <content_attribute id="language-discrimination">none</content_attribute>
+    <content_attribute id="social-chat">none</content_attribute>
+    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-audio">none</content_attribute>
+    <content_attribute id="social-location">none</content_attribute>
+    <content_attribute id="social-contacts">none</content_attribute>
+    <content_attribute id="money-purchasing">none</content_attribute>
```

Comment:
This seems inaccurate. Is Picard no longer asking people to donate to MetaBrainz?

Reply:
I checked with OARS asking for money donation is not same as money purchasing. Hence, I marked it as none.

Reply:
Alright. The generator at https://hughsie.github.io/oars/generate.html has "Users are encouraged to donate real money, e.g. using Patreon" (which Picard *does* do) as `money-purchasing` = `mild`. But if you've asked OARS people and they say differently, then that's probably correct. :)

Reply:
@Freso Sorry for the error on my side. It indeed needs to be mild. It was a miscommunication on my part. I will correct it. I didn't know the donation column was in In-App Purchases.
```suggestion
    <content_attribute id="money-purchasing">mild</content_attribute>
```

Reply:
Does the encouragement to donate appear inside the app?

Reply:
Yes it does on the About Page in the application.

Reply:
We have a note in the about page and an actual "Donate..." menu entry, so I think setting this to mild would be just fair. Let's do this.